### ISSUE TITLE
Postgresql app development

### ISSUE THREAD

**Issue by jarpineh** [craigkerstiens/postgresguide.com#9]
Hi.

I think the Postgres guide is a great idea. I am just starting to use Postgres for web apps. I have been using it mostly through Django ORM, but it would be good to know your database well. Especially since Postgres is really an application platform on top of a database, with its support for languages other than SQL and interface for external dbs etc. And relational model is only one of the available data models.

I have tried searching about this, but mostly in vain. I'd like to see the Postgres Guide explain how to put up a development workflow with source code management, testing, deployment and so on. What tools are there to interact with the database for changing schema, editing different types of code with syntax highlight. Everything should be in version control.

I have the Posgres administration cookbook and high performance book, but neither really talk about development. Articles I have found mostly speak about SQL coding in general.

I read Craig Kerstiens' blog posts about notable Postgres features, most of which I knew nothing about. I have use cases for them, like access to Redis and listen/notify. What I lack is knowledge on how to develop with Postgres. Right now it seems the only approach is to use Pgadmin3 as an IDE (which keeps crashing) or the psql cli. Since my knowledge on this is severely limited, I can't yet contribute to the guide all that much.

tl:dr; I'd like to know what is the Postgres way of app development, what are the best practises and tools.
